.github/workflows/docat.yml

```diff
@@ -81,7 +81,7 @@ jobs:
       - uses: actions/checkout@v4
       - name: Build Image
         run: |
-          docker build . --tag ${{ matrix.registry.name }}/${{ matrix.registry.org }}/docat:${{ github.sha }}
+          docker build . --build-arg DOCAT_VERSION=$(git describe --tags --exact-match HEAD 2>/dev/null || git rev-parse --short HEAD) --tag ${{ matrix.registry.name }}/${{ matrix.registry.org }}/docat:${{ github.sha }}
```

Comment:
Another option would be to use `git describe --tags --always`. The advantage would be that 
one sees directly 'how far away' from the last tag we are:

`TAG-N_COMMITS_FROM_TAG-HASH`

Reply:
Ah that's what that number's for. I thought about it, but I didn't like that additional `1` in `1.1.1-1-ge6e98ed`.
In that case I think I'll switch to using that, it would simplify the code as well which is always a benefit :+1:

Reply:
@randombenj Done